source/settings.cpp

```diff
@@ -669,6 +671,14 @@ namespace Settings {
   Option ColoredBossKeys = Option::Bool("Colored Boss Keys",  {"Off", "On"}, {coloredBossKeysDesc}, OptionCategory::Cosmetic);
   Option MirrorWorld =     Option::Bool("Mirror World",       {"Off", "On"}, {mirrorWorldDesc},     OptionCategory::Cosmetic);
 
+  static std::vector<std::string> fanfareOptions = {"Off", "Only Fanfares", "Fanfares + Ocarina Music"};
+  static std::vector<std::string_view> fanfareDescriptions = {fanfaresOffDesc, onlyFanfaresDesc, fanfaresOcarinaDesc};
+
+  Option ShuffleMusic =    Option::Bool("Shuffle Music",           {"Off", "On"},    {""},                  OptionCategory::Cosmetic);
+  Option ShuffleBGM =      Option::Bool("  Shuffle BGM",           {"Off", "On"},    {""},                  OptionCategory::Cosmetic);
+  Option ShuffleFanfares = Option::U8  ("  Shuffle Fanfares",      {fanfareOptions}, {fanfareDescriptions}, OptionCategory::Cosmetic);
+  Option ShuffleOcaMusic = Option::Bool("  Shuffle Ocarina Music", {"Off", "On"},    {""},                  OptionCategory::Cosmetic);
```

Comment:
Also just noticed this, can you add descriptions for the options? They don't have to be super in-depth, but they should exist. We usually just copy descriptions from OoTR if you can't think of anything to write. For example, the description for BGM from OoTR is just "Area background music is randomized."

Reply:
Done